src/main/java/com/commercetools/http/converter/json/PrettyFormattedBody.java

```diff
@@ -0,0 +1,34 @@
+package com.commercetools.http.converter.json;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Class to wrap a response body if (JSON) pretty formatting could be expected in the result output.
+ */
+public final class PrettyFormattedBody {
+    private final Object body;
+    private final boolean pretty;
+
+    private PrettyFormattedBody(@Nonnull final Object body, final boolean pretty) {
+        this.body = body;
+        this.pretty = pretty;
+    }
+
+    /**
+     * @return real response body to output
+     */
+    public Object getBody() {
+        return body;
+    }
+
+    /**
+     * @return <b>true</b> if the {@link #body} should be pretty printed in the response.
+     */
+    public boolean isPretty() {
```

Comment:
no, i want to introduce them as public API of this class, so any other class in the future could verify the body and formatting type. 
As you know already - i'm package-private hater since (IMHO) it goes agains common OOP definition (but still could accept package-private scope as a workaround for internal tests!)